core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -291,6 +291,21 @@ class TaskSetManagerSuite
     assert(manager.resourceOffer("execA", "host1", ANY)._1.get.index === 0)
   }
 
+  test("skip unsatisfiable locality levels (the case TaskLocation is HostTaskLocation)") {
```

Comment:
Maybe we should add `SPARK-37488` to the test name as prefix

Reply:
Thanks for the reminder, I'll add it later